src/System.Security.Cryptography.Algorithms/tests/System.Security.Cryptography.Algorithms.Tests.csproj

```diff
@@ -1,5 +1,8 @@
-﻿<?xml version="1.0" encoding="utf-8"?>
+<?xml version="1.0" encoding="utf-8"?>
 <Project ToolsVersion="12.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <PropertyGroup>
+    <Configuration Condition="'$(Configuration)'==''">Windows_Debug</Configuration>
+  </PropertyGroup>
   <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory), dir.props))\dir.props" />
   <PropertyGroup>
     <Configuration Condition=" '$(Configuration)' == '' ">Debug</Configuration>
```

Comment:
I'm a bit confused.  Where does the configuration come from now?  It used to be (as far as I understood) that dir.props interpreted OSGroup (et al) and set Configuration.  Now you're setting configuration to Windows_Debug before calling dir.props; but there's still an "if it's unset set it" here.

My main comment is "I think this copy of the default value populator can be removed", but when writing it I realized I'm now confused :).

This likely applies to other projects, I skipped down to what I was most familiar with to understand the impact.